core/templates/pages/preferences-page/preferences-page.component.html

```diff
@@ -23,10 +23,11 @@
                  ng-click="$ctrl.showEditProfilePictureModal()">
             </div>
             <i class="material-icons oppia-editor-edit-icon" style="top: 8px; right: 8px;">&#xE254;</i>
-            <img ng-if="$ctrl.profilePictureDataUrl"
-                 ng-src="<[$ctrl.profilePictureDataUrl]>"
-                 class="img-thumbnail protractor-test-custom-photo">
-            <img ng-if="!$ctrl.profilePictureDataUrl" ng-src="<[$ctrl.getStaticImageUrl('/general/no_profile_picture.png')]>" class="img-thumbnail">
+            <picture>
+              <source type="image/webp" ng-srcset="<[$ctrl.getStaticImageUrl('/general/no_profile_picture.webp')]>">
+              <source type="image/png" ng-srcset="<[$ctrl.getStaticImageUrl('/general/no_profile_picture.png')]>">
+              <img ng-if="!$ctrl.profilePictureDataUrl" ng-src="<[$ctrl.getStaticImageUrl('/general/no_profile_picture.png')]>" class="img-thumbnail">
+            </picture>
```

Comment:
You have changed the logic here a bit, you need one `<picture>` for the case when the profile image is not defined and one for the case when the profile image is defined.